### PR TITLE
Gazelle flatmap pop ups

### PR DESCRIPTION
**Pull request by Tehsurfer**
I'm currently modifying the flatmap tooltips to match Domics desgins:

image

Data point click: 
https://share.goabstract.com/cba13dfa-c5d2-4265-a15a-727386996599

Nerve click:
https://share.goabstract.com/bf22593a-2dce-41bc-af32-d71e9467df71

The method uses a technique for turning css traingles into a triangular
border found here:

https://stackoverflow.com/a/9450768/2217801

It's been modified to place the border triangle at the center of the
parent div (>>>.flatmap-tooltip-popup .mapboxgl-popup-content)

### CODE REVIEW

**Comment by Tehsurfer** [comment]
Testing across browsers:

### Windows

Firefox:
image

Chrome:
image

Edge:
ietest

**Comment by Tehsurfer** [comment]
Update: Chrome seems to often have this happen:
image

Not sure why at this stage

**Comment by alan-wu** [comment]
It is working fine on Chrome for me.

File: src/components/FlatmapVuer.vue [hunk 1]
```diff
@@ -199,6 +199,8 @@ export default {
         const taxonomy = this.entry;
         const data = { taxonomy: taxonomy, resource: resource, label: label,
           feature: feature, userData: args, eventType: eventType};
+        console.log(data.resource, data.feature)
+        window.daaaata = data
```
Comment (agarny) [inline src/components/FlatmapVuer.vue:203]: Is it really meant to be `daaaata`?

**Comment by Tehsurfer** [comment]
Closing this as it is now in main via upstream downstream popups

**Pull request closed without merging**